src/Microsoft.Health.Fhir.TemplateManagement/Models/ArtifactImage.cs

```diff
@@ -11,10 +11,10 @@ namespace Microsoft.Health.Fhir.TemplateManagement.Client
 {
     public class ArtifactImage
     {
-        public ImageInfo Info { get; set; }
+        public string ImageDigest { get; set; }
```

Comment:
The namespace of this file is inconsistent with its directory (`.Models` instead of `.Clients`).